tests/e2e/k8s/snapshotter-cri.yaml

```diff
@@ -187,3 +202,36 @@ data:
     [image]
     public_key_file = ""
     validate_signature = false
+
+  nydusd-config.json: |-
```

Comment:
Ditto, maybe we can reuse the `nydusd-config-xxx.json`.

Reply:
Fixed.